pio_examples/gui/src/gui.ino

```diff
@@ -258,7 +261,15 @@ void setup(void)
      * password, for example begin("ESPUI Control", "username", "password")
      */
     ESPUI.sliderContinuous = true;
-    ESPUI.begin("ESPUI Control");
+    ESPUI.prepareFileSystem();
```

Comment:
Have we actually checked how much time and space this needs ? originally I had this uploaded this in a first sketch and then removed it so there is less memory used for the actual programm... but this might be all outdated anyways...

Reply:
For the example, it did not use a lot. Normally in pio you use the "Build and upload filesystem" command but that requires a bit of tweaking in the PIO settings and you need to make the generation of a data directory automatic. Not hard, just a few extra steps that users tend to forget to do.